litmus-portal/backend/graphql-server/pkg/k8s/client/dynamic/incluster.go

```diff
@@ -0,0 +1,31 @@
+package dynamic
```

Comment:
Do we need this in the GQL Server?

Reply:
Yes, this pkg will be used by the other microservices like agent/selfdeployer/graphqlserver